arcgis-ios-sdk-samples/Maps/Create and save a map/CreateSaveMapViewController.swift

```diff
@@ -52,8 +54,31 @@ class CreateSaveMapViewController: UIViewController, CreateOptionsViewController
         AGSAuthenticationManager.shared().oAuthConfigurations.add(config)
         AGSAuthenticationManager.shared().credentialCache.removeAllCredentials()
         
-        // Initially show the map creation UI.
-        performSegue(withIdentifier: "CreateNewSegue", sender: self)
+        // Temporarily unset the API key for this sample.
+        // Please see the additional information in the README.
+        let apiKey = AGSArcGISRuntimeEnvironment.apiKey
+        AGSArcGISRuntimeEnvironment.apiKey = ""
+        
+        let portal = AGSPortal(url: URL(string: "https://www.arcgis.com")!, loginRequired: true)
+        self.portal = portal
+        portal.load { [weak self] (error) in
+            guard let self = self else { return }
+            if let error = error {
+                print(error)
```

Comment:
I'm a bit unsure here. We shouldn't print anything to console. What about presentAlert or just silently fail here?